redisson/src/main/java/org/redisson/spring/cache/RedissonCache.java

```diff
@@ -238,6 +242,12 @@ long getCacheMisses(){
     long getCachePuts() {
         return puts.get();
     }
+
+    Long getCacheEvictions() {
+        return evictions != null
```

Comment:
evictions can't be null

Reply:
It is null, when this cache is backed by RMap (created with: `public RedissonCache(RMap<Object, Object> map, boolean allowNullValues)` constructor)

Reply:
this is why I asked to init it in the field